packages/editor-ui/src/components/VariablesRow.vue

```diff
@@ -53,51 +88,80 @@ async function onSave() {
 
 async function onEdit() {
 	emit('edit', modelValue.value);
+
+	await nextTick();
+
+	focusFirstInput();
 }
 
 async function onDelete() {
 	emit('delete', modelValue.value);
 }
 
+function onValidate(key: string, value: boolean) {
+	formValidationStatus.value[key] = value;
+}
+
 function onUsageClick() {
 	copyToClipboard(usage.value);
 	showMessage({
 		title: i18n.baseText('variables.row.usage.copiedToClipboard'),
 		type: 'success',
 	});
 }
+
+function focusFirstInput() {
+	keyInputRef.value?.inputRef?.focus?.();
+}
 </script>
 
 <template>
 	<tr :class="$style.variablesRow">
-		<td>
-			<span v-if="!editing">{{ data.key }}</span>
-			<n8n-input
-				v-else
-				data-test-id="variable-row-key-input"
-				:required="true"
-				:placeholder="i18n.baseText('variables.editing.key.placeholder')"
-				v-model="modelValue.key"
-				ref="keyInputRef"
-			/>
+		<td class="variables-key-column">
```

Comment:
The most common use case is to make non-obfuscated class names available in another component. 

For example, if we want to style the VariablesRow in a specific usage context, we'll be able to modify the classes we expose. Here, the choice was made so that we're able to modify the classes for both the `thead tr th` and the `tbody tr td` inside of the VariablesView.